Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/CertificateStateHelper.kt

```diff
@@ -0,0 +1,140 @@
+package de.rki.coronawarnapp.util
+
+import android.content.Context
+import android.view.View
+import android.widget.TextView
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.covidcertificate.common.certificate.CwaCovidCertificate
+import de.rki.coronawarnapp.covidcertificate.recovery.core.RecoveryCertificate
+import de.rki.coronawarnapp.covidcertificate.test.core.TestCertificate
+import de.rki.coronawarnapp.covidcertificate.vaccination.core.VaccinationCertificate
+import de.rki.coronawarnapp.databinding.IncludeCertificateQrcodeCardBinding
+import de.rki.coronawarnapp.util.ContextExtensions.getDrawableCompat
+import de.rki.coronawarnapp.util.TimeAndDateExtensions.toShortDayFormat
+import de.rki.coronawarnapp.util.TimeAndDateExtensions.toShortTimeFormat
+import de.rki.coronawarnapp.util.TimeAndDateExtensions.toUserTimeZone
+
+object CertificateStateHelper {
+
+    fun IncludeCertificateQrcodeCardBinding.bindValidityViews(
+        context: Context,
```

Comment:
you can access the context from the view 😉

Reply:
I've tried this but I think it only works when it's an extension function over an actual View like `TextView`. This one is an extension function over the binding of this `include` and I don't see any context if I remove it explicitly.

Reply:
maybe `val context = root.context` ?>